src/storage/AccountStorageProvider.tsx

```diff
@@ -399,6 +402,23 @@ const useAccountStorageHook = () => {
     ],
   )
 
+  useEffect(() => {
+    const connection = getConnection(cluster)
+    if (connection) {
+      cache?.close()
+      setCache((c) =>
+        !c
+          ? new AccountFetchCache({
```

Comment:
So what is `AccountFetchCache`? I'm a little confused on the different concepts of Account. We have `useAccount`, `useAccountStorage`, `useIdlAccount`, `CSAccount`, etc